hawc/apps/summary/views.py

```diff
@@ -445,7 +445,28 @@ def get_context_data(self, **kwargs):
             len(context["breadcrumbs"]) - 1, get_visual_list_crumb(self.assessment)
         )
         if context["visual_type"] == constants.VisualType.PRISMA:
-            context.update(prisma_data=models.Visual.get_prisma_data(self.assessment))
+            context.update(api_url=reverse("summary:api:visual-data-json"))
+        context["config"] = schemas.VisualConfig(
```

Comment:
the config that's being sent to the template is completely different than the config that's being passed to get the dataset, I would not overload the two.  For this PR, I'd focus exclusively on the config required to get a dataset.